content/numpy/concepts/math-methods/math-methods.md

```diff
@@ -0,0 +1,71 @@
+
+
+# Math Methods
+  - Numerical Python abreviated as numPy, used for numerical computing.
+  - It provides a wide range of mathematical functions and operations on scalar values, arrays and matrices.
+  - Acts as a fundamental tool for scientific computing, data analysis, and machine learning.
+  - NumPy's math methods facilitate complex mathematical computations efficiently, making it indispensable in various domains including scientific research, engineering, finance, and data analysis.
+  - Understanding these methods empowers practitioners to leverage the full potential of NumPy for their numerical computing tasks.
+## Syntax
+
+ numpy. <math_method>
+
+## Example
+![image](https://github.com/gitsoft145/docs/assets/78934620/7f0cf44a-f91d-4ef2-95e5-7cb065387980)
+
+
+## Output
+![image](https://github.com/gitsoft145/docs/assets/78934620/cb0b3d5b-7c91-45a5-b301-3d56cce64bb0)
+
+
+
+## Basic Categories of Math Methods
+
+  ### 1. Basic Arithmetic Operations
+  
+      - Addition (numpy.add): Element-wise addition of arrays and scalar values.
+      - Subtraction (numpy.subtract): Element-wise subtraction of arrays.
+      - Multiplication (numpy.multiply): Element-wise multiplication of arrays.
+      - Division (numpy.divide): Element-wise division of arrays.
+      - Power (numpy.power): Element-wise exponentiation of arrays.
+      
+  ### 2. Trigonometric Functions
+  
+      - Sine (numpy.sin): Computes the sine of each element in the array.
+      - Cosine (numpy.cos): Computes the cosine of each element in the array.
+      - Tangent (numpy.tan): Computes the tangent of each element in the array.
+      - Inverse Sine (numpy.arcsin): Computes the inverse sine of each element in the array.
+      - Inverse Cosine (numpy.arccos): Computes the inverse cosine of each element in the array.
+      - Inverse Tangent (numpy.arctan): Computes the inverse tangent of each element in the array.
```

Comment:
Only two spaces are needed before every point for the bullets.
For all the points listed above, wrap the syntax in the backticks. Example:
   - Sine `numpy.sin()`: Computes the sine of each element in the array.